front/components/items/Appointment.tsx

```diff
@@ -32,12 +44,17 @@ const Appointment = () => {
                 $eq: queryChosenDate,
             };
         }
+        filters.therapist = {
+            first_name: {
+                $eq: chosenTherapist,
+            },
+        };
         strapiAppointmentGet(filters).then((res) => {
             setAppointmentsTherapist(res.data.data);
```

Comment:
I cannot understand why do you have same values in appointmentsTherapist and appointmentsDate. Considering by names the variables should store different values